style.css

```diff
@@ -45,26 +42,50 @@ html {
 }
 
 .hero-banner-container {
-  position: relative;
-  height: 50rem;
-  overflow: hidden;
-  background: url(https://dispatch.barnesandnoble.com/content/dam/ccr/homepage/daily/2022/02/01/Gateway_Billboard-C_OMP.jpg)
-    no-repeat center;
-  background-size: cover;
-}
-
-.hero-image {
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  overflow: hidden;
-  text-align: center;
-  text-transform: uppercase;
+  min-height: 10rem;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 1.5rem;
+  flex-wrap: wrap;
+  margin-top: 2rem;
```

Comment:
You could use the shorthand for margin to avoid extra line of code.

e.g. 
`margin: 2rem initial 1.5rem initial`
which goes like 
`margin: [top] [right] [bottom] [left]`